src/pages/components/textArea/Textarea.module.scss

```diff
@@ -0,0 +1,7 @@
+.textArea {
+  padding: 10px;
```

Comment:
nesse artigo do @Alecell sobre unidades de medidas, ele fala sobre sempre usar unidades relativas como `rem` em padding, e nunca medidas fixas como px. O arquivo `base.scss` onde é configurando o `font-size` do body como `62,5%` para poder trabalhar com rem ainda não foi mergeado mas não precisa se preocupar com isso porque o pr já ta pronto, só to aguardando uma revisão

Reply:
artigo citado: https://www.linkedin.com/pulse/unidades-css-um-guia-completo-alexandre-gomes/